src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -202,8 +202,8 @@ Result<std::string> get_full_path(const std::string &filename,
         return file_path;
     } else {
         // If this is `lpython`, do a special lookup
-        if (filename == "lpython.py") {
-            file_path = runtime_library_dir + "/lpython/" + filename;
+        if (filename == "lpython.py" || filename == "lptypes.py") {
+            file_path = runtime_library_dir + "/lpython/" + "lpython.py";
```

Comment:
Let's remove `lptypes.py`, let's only support lpython.py.

Reply:
>Let's remove lptypes.py, let's only support lpython.py.

Got it.

Since we are currently unable to upload `lpython.py` as an `lpython` package, I will include it (and its dependencies) as module(s) inside every package.

Reply:
>I will include it (and its dependencies) as module(s) inside every package.

Or maybe we need not include them. I just attempted uploading the `lpynn` package without including the `lpython.py` (and its dependency file) and it seems to upload.